front/src/modules/ui/themes/colors.ts

```diff
@@ -104,6 +104,7 @@ export const color = {
   orange30: '#f09797',
   orange20: '#fbc5c5',
   orange10: '#fde4e4',
+  // TODO: Why color are not matching with design?
```

Comment:
The design is changing often, ask @Bonapara directly for that.

Reply:
Haha, so @Bonapara is it normal ?